numpy/lib/shape_base.py

```diff
@@ -517,22 +517,28 @@ def expand_dims(a, axis):
     Insert a new axis that will appear at the `axis` position in the expanded
     array shape.
 
-    .. note:: Previous to NumPy 1.13.0, neither ``axis < -a.ndim - 1`` nor
-       ``axis > a.ndim`` raised errors or put the new axis where documented.
-       Those axis values are now deprecated and will raise an AxisError in the
-       future.
-
     Parameters
     ----------
     a : array_like
         Input array.
-    axis : int
-        Position in the expanded axes where the new axis is placed.
+    axis : int or tuple of ints
+        Position in the expanded axes where the new axis (or axes) is placed.
+
+        .. deprecated: 1.13.0
+
+        Passing an axis where ``axis > a.ndim`` will be treated as
```

Comment:
I think these should be indented, right now it seems not rendered at all: https://10461-908607-gh.circle-artifacts.com/0/home/circleci/repo/doc/build/html/reference/generated/numpy.expand_dims.html?highlight=expand_dims#numpy.expand_dims

Reply:
Done, thanks!